src/Joomlatools/Console/Command/Site/Configure.php

```diff
@@ -81,6 +96,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->_promptDetails($input, $output);
         }
 
+        $options = $input->getOption('options');
+        if ($options !== null)
+        {
+            if (!file_exists($options)) {
+                throw new Exception(sprintf('Additional option file \'%s\' does not exist', $options));
+            }
+
+            $contents = file_get_contents($options);
+
+            try {
+                $this->_extra_options = Yaml::parse($contents);
```

Comment:
We decided to only support one format (YAML), as that will also be used for a global configuration file as planned in joomlatools/joomlatools-console#12.